templates/data/incubator-buildpack-data.yaml

```diff
@@ -204,16 +204,48 @@ templates:
                     - milv_jobConfig
       - to: ../prow/jobs/incubator/reconciler/reconciler.yaml
         localSets:
+          command_integration:
+            annotations:
+              pipeline.clusterprovisioning: kyma cli
           reconciler_jobConfig:
+            command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build-generic.sh"
             args:
               - "/home/prow/go/src/github.com/kyma-incubator/reconciler"
             env:
               GO11MODULE: "on"
             branches:
               - ^main$
+          reconciler_e2e_jobConfig:
+            command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/reconciler-e2e-gardener.sh"
+            args:
+              - "/home/prow/go/src/github.com/kyma-incubator/reconciler"
+          reconciler_gardener_e2e:
+            labels:
+              preset-gardener-gcp-kyma-integration: "true"
+              preset-kyma-cli-stable: "true"
+              preset-cluster-version: "true"
+            env:
+              KYMA_PROJECT_DIR: "/home/prow/go/src/github.com/kyma-project"
+              GARDENER_REGION: "europe-west4"
+              GARDENER_ZONES: "europe-west4-b"
+            annotations:
+              pipeline.platform: gardener_gcp
         jobConfigs:
           - repoName: kyma-incubator/reconciler
             jobs:
+              - jobConfig:
+                  name: pre-main-kyma-incubator-reconciler-e2e
+                  run_if_changed: "^resources"
+                  image: "eu.gcr.io/kyma-project/test-infra/pr/kyma-integration:PR-4084-k8s1.18"
```

Comment:
Update image once this [PR](https://github.com/kyma-project/test-infra/pull/4084) is approved and we have a new image for kyma-integration.

Reply:
updated!